docs/content/en/docs/reference/clusterspec/vsphere.md

```diff
@@ -18,55 +18,55 @@ The following additional optional configuration can also be included:
 * [Registry Mirror]({{< relref "optional/registrymirror.md" >}})
 
```

Comment:
I guess relrefs won't work in this context and we are going full html to do this?

Reply:
Yes. Once you put ````yaml` or equivalent around the content, you can't add links or other markup. We lose the "copy-to-clipboard" button, so someone would have to highlight to copy the whole text themselves. But I think they wouldn't need to do that since they would be working from their own generated yaml file.